build.gradle

```diff
@@ -7,10 +7,15 @@ plugins {
     id "org.cyclonedx.bom" version "1.8.2"
 }
 
-version =  getGitVersion()
+if (project.hasProperty('isTagTriggered')) {
+    version = getAbbreviatedGitVersion()
+} else {
+    version = getGitVersion()
+}
 
 ext {
     releaseVersion = getReleaseVersion()
+    println(releaseVersion)
```

Comment:
just double checking but is this testing remains or do we want this as part of hte log?